### DISCUSSION TITLE
Deviating from the folder structure for long-running series?

### DISCUSSION THREAD

**Discussion by Neurrone** [seanap/Plex-Audiobook-Guide#57] [Q&A]
Hi,

There are many books in e.g, star wars or star trek. I find that for those types of books, I prefer to have a overall series title first (e.g, "star wars"), then another classification (e.g, "new republic era"), then a chronological list of books within that era.

1. Will there be any ill effects for Plex, Booksonic or other audiobook management software if I deviate from the usual author / series / book structure?
2. Another question is how this info could be captured in id3 tags. As far as I know, there isn't a standard way to put this, so I'm unsure if there's any point in writing those to tags.

**Comment by seanap** [comment]
1. Not for plex, but booksonic relies on the folder structure.  I don't want to say plex doesn't care at all about folder structure but in my testing as long as the ALBUM and ARTIST tags are set then the folder structure doesn't matter the agent will match the books, if they are not set then Plex does fall back to looking for the info from folder names but the agent automatch will fail.   For booksonic if for example you change the folder structure from /author/series/book/...  To /series1/author/series2/book/... Then booksonic would classify series1 as an author.  Plex does not handle series very well, best we can do is shove the series as the "Mood" tag in plex, so you can have books with multiple mood (series) tags.  
2. Honestly the star wars series pretty much broke me and I gave up and created custom actions and modified the src script just for those series.  The only way by tag to have multiple series type tags is to create custom tags.  Eg. SERIES1/SERIES1-PART and SERIES2/SERIES2-PART.  The problem with this is there are no programs that will read those tags.  It may be best to manually create collections in Plex to handle multi series.
